src/SourceBuild/patches/runtime/0001-Fix-zlib-ng-build-failure-caught-in-the-sdk-repo.patch

```diff
@@ -0,0 +1,62 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Carlos=20S=C3=A1nchez=20L=C3=B3pez?=
+ <1175054+carlossanlop@users.noreply.github.com>
+Date: Thu, 11 Jul 2024 11:28:54 -0700
+Subject: [PATCH] Fix zlib-ng build failure caught in the sdk repo. Fixed by
+ converting its NativeLibrary entry in Microsoft.NETCore.Native.Unix.targets
+ to a NetCoreAppNativeLibrary, and place it after
+ System.IO.Compression.Native, to ensure the linker detects the symbols, since
+ it is unable to look back.
+
+Co-authored-by: Adeel Mujahid <3840695+am11@users.noreply.github.com>
```

Comment:
Please add a `Backport: <github issue/pr link>` line.  This helps track when this patch can be removed.

See https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines

Reply:
```suggestion
Co-authored-by: Adeel Mujahid <3840695+am11@users.noreply.github.com>

Backport: https://github.com/dotnet/runtime/pull/104750
```